CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## [3.0.4+portage-3.0.5]
+- Fixed pdf download issue
```

Comment:
We need to add this on the CHANGELOG under a new version. 
Also, we need to be under a subheading for FIXED.
Now that we are keeping better track of issues we should start adding links to the issue being fixed #153 

You can see[ jupiter's CHANGELOG](https://github.com/ualbertalib/jupiter/blob/master/CHANGELOG.md) as an example

Reply:
Working on it : ) should I put today as the release date?

Reply:
Yes please

Reply:
Yes please